analytics/docker-compose.yml

```diff
@@ -2,6 +2,16 @@ version: '3'
 
 services:
 
+  grants-analytics-db:
+    image: postgres:14-alpine
+    container_name: grants-analytics-db
+    command: postgres -c "log_lock_waits=on" -N 1000 -c "fsync=off"
+    env_file: ./local.env
+    ports:
+      - "5432:5432"
```

Comment:
Two suggestions -
1. Postgres 15 is what I think we're running in AWS, so probably should match that here - for the API docker-compose, I'm actually going to send a PR out shortly for that as I want a Postgres 15 feature for something
2. Should we run on a different port than the API? Otherwise you can't run both at the same time